### PR TITLE
feat: move alerts activity to alerts page

### PR DESCRIPTION
**Pull request by loriepisicchio**
## Issue

https://gravitee.atlassian.net/browse/APIM-1141

## Description

Move alerts history into alerts page.

⚠️  AngularJS code inside

screenshot-localhost_3000-2023 03 24-11_12_32
screenshot-localhost_3000-2023 03 24-11_12_45
screenshot-localhost_3000-2023 03 24-11_13_00

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-aotodaenkj.chromatic.com)

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.z6.web.core.windows.net/apim-1141-move-alerts-dashboard-to-alerts-page/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

### CODE REVIEW

File: gravitee-apim-console-webui/src/components/alerts/alerts.component.ts [hunk 1]
```diff
@@ -48,7 +48,11 @@ const AlertsComponent: ng.IComponentOptions = {
           alertId: alertId,
         });
       } else {
-        $state.go('management.alerts.' + suffixState, { alertId: alertId });
+        if (suffixState === 'alertnew') {
```
Comment (jourdiw) [inline gravitee-apim-console-webui/src/components/alerts/alerts.component.ts]: Maybe better to have the default be to create a new alert?

File: gravitee-apim-console-webui/src/management/configuration/notifications/global.notifications.settings.route.ts [hunk 2]
```diff
@@ -56,11 +56,29 @@ function applicationsNotificationsRouterConfig($stateProvider) {
...
     .state('management.alerts', {
-      abstract: true,
       url: '/alerts',
+      template: require('../../../components/alerts/alertTabs/alert-tabs.html'),
+      controller: 'AlertTabsController',
+      controllerAs: '$ctrl',
+    })
+    .state('management.alerts.activity', {
+      url: '/activity',
+      template: require('../../dashboard-ajs/alerts-dashboard/platform-alerts-dashboard.html'),
```
Comment (jourdiw) [inline gravitee-apim-console-webui/src/management/configuration/notifications/global.notifications.settings.route.ts]: WDYT if the files are moved to /components/alerts and renamed "Activity"?

**Review by jourdiw — APPROVED** [review]

**Comment by jourdiw** [comment]
Also, change title "Alerts" --> "My Alerts" & "Alerts board" --> "Activity"

**Review by ThibaudAV — APPROVED** [review]
Good for me but I wonder how to inform the user? In the release note? I wonder the same for all the transformation of the Home page. 
and I don't know if there are any updateable captures in the doc or others

**Comment by phiz71** [comment]
> Good for me but I wonder how to inform the user? In the release note? I wonder the same for all the transformation of the Home page. and I don't know if there are any updateable captures in the doc or others

For me, the release note is the best place to inform the user.

We could also do as we did when we moved the "design" submenu item on top of the submenu, by adding a small note
image

WDYT? Would it be hard to implement?

**Pull request merged**